FTP passive mode and fixed broken links

grimctl opened the pull request.

Commit:
Added FTP install for windows doc

CountBacula committed:
Added "windows" keyword

Commit:
Updated FTP Install file location

Commit:
Added FTP passive mode for windows doc

Commit:
Fixed broken firewall links

Commit:
Added meta data

Commit:
Added passive mode to index

grimctl committed:
Merge branch 'master' into master

Commit:
Merge remote-tracking branch 'ukfast/master'

Commit:
Added FileZilla Client doc and fixed grammar errors

Commit:
Added reset password in ad doc

Commit:
Merge remote-tracking branch 'ukfast/master'

CountBacula commented on source/operatingsystems/windows/activedirectory/resetpassword.md:
```diff
@@ -0,0 +1,43 @@
+![AD reset](files/resetpassword/AD_reset.PNG)
+
+- Within this window, enter a `New password` for the user.
+- If you would like the user to change the password upon next logging on, check the `User must change password at next logon` check box.
```
This may cause problems when they attempt to connect over RDP with NLA enabled (which should be always). I would recommend UKFast client's always clear this checkbox but instruct users to update their password after they've logged in via ctrl+alt+insert etc.

grimctl replied:
fc9c54a8f13df22c578962e250349633f9516b0c

CountBacula commented on source/operatingsystems/windows/activedirectory/resetpassword.md:
```diff
@@ -0,0 +1,43 @@
+
+This guide will show you how you can reset a user's password with Active Directory, you will need to gain access to the Domain Controller within your network to follow this guide.
+
+I will be providing two examples of resetting a user's password, one using the `Active Directory Users and Computers` tool and the other using `Powershell`.
```
Power**S**hell, here and throughout.

CountBacula commented on source/operatingsystems/windows/ftp/filezilla_client.md:
```diff
@@ -0,0 +1,75 @@
+  - **Desktop Icon** - This will add the FileZilla Client desktop icon to your desktop.
+- Most users will install the components that are selected by default.
+
+After selecting the desired components to install, click the `Next` button and a window where you can choose a location on your server to store the application files will be displayed.
```
server/workstation etc.

CountBacula commented on source/operatingsystems/windows/ftp/filezilla_client.md:
```diff
@@ -0,0 +1,75 @@
+# Connecting to an FTP Server using FileZilla Client
+
+This guide will assist you in connecting to your FTP server using the FileZilla Client.
```
Add a note on usage and reference to guide on installing server and configuring users to connect as?

CountBacula commented on source/operatingsystems/windows/ftp/passive_mode.md:
```diff
@@ -0,0 +1,71 @@
+
+## FileZilla Server
+
+Once you have installed and configured FileZilla Server on your Windows Server and you would like to enable FTP passive ports, you will first need to open the `FileZilla administrative intereface`.
```
interface -typo, here and below also

CountBacula commented on source/operatingsystems/windows/ftp/passive_mode.md:
```diff
@@ -0,0 +1,71 @@
+
+- Once you have opened IIS, please click on your `Server`, which will be located on the left-hand side of the IIS window.
+
+You will then presented with a similar window to the below image:
```
missing word "be"

CountBacula reviewed (changes requested):
Number of typos and I've added some suggestions - see in-line comments.

Commit:
Removed point to check the User must change password at next logon check box

Commit:
Merge remote-tracking branch 'upstream/master'

darrentayloruk committed:
Updating the website styles to give this a look consistent with other UKFast sites - also structure tidyup (#615)

* Rename cloud to ecloud to keep branding
* Rename Domains to domain to keep consistency
* Adding in redirects into /conf for the above
* Layout update to refresh the site look and feel and be on brand
* Updated footer links to match live site

laf committed:
Updated files in support of BAME. Hopefully the use of these new terms still makes sense in the context of the documents (#614)

LGTM

darrentayloruk committed:
Correcting links so they don't rely on the redirect (#617)

* Rename cloud to ecloud to keep branding
Rename Domains to domain to keep consistency
Adding in redirects into /conf for the above
Layout update to refresh the site look and feel and be on brand
Updated footer links to match live site

* Adding meta

* More meta

* more meta

* Removing unused conf files
Correcting redirect

* Correcting lowercase domain

* Correcting links to go to correct place and not rely on the redirect

* Appeasing meta checker

* Remove dupe meta

* meta correction

darrentayloruk committed:
CI Fixes (#619)

* Rename cloud to ecloud to keep branding
* Rename Domains to domain to keep consistency
* Adding in redirects into /conf for the above
* Layout update to refresh the site look and feel and be on brand
* Updated footer links to match live site
* Removing unused conf files
* Correcting links to go to correct place and not rely on the redirect

darrentayloruk committed:
Nginx redirects to support PR#614 (#620)

* Adding redirects to go with PR614

ledzepp4eva committed:
updating typo in duply wiki (#537)

* Creating a help page for testing your domain before enabling DDoSx

This page will be linked in the on boarding process of ddosx to help clients to test domains via DDoSx before putting them live in case of issues with going through the platform.

* changing yum command to fix typo
  The correct command for enabling a repo is --enablerepo not --enablrepo
* Tidying up the DDoSx branding name and formatting
* Cleaning up formatting and adding in Metadata
* Move meta further down the page

Co-authored-by: Liam Moncur <liam.moncur@ukfast.co.uk>
Co-authored-by: Matt Calvert <miff2000@users.noreply.github.com>

miff2000 committed:
Correct Note: syntax be restructuredtext

darrentayloruk committed:
Homepage fixes (#621)

* Rename cloud to ecloud to keep branding
Rename Domains to domain to keep consistency
Adding in redirects into /conf for the above
Layout update to refresh the site look and feel and be on brand
Updated footer links to match live site

* Adding meta

* More meta

* more meta

* Removing unused conf files
Correcting redirect

* Correcting lowercase domain

* Correcting links to go to correct place and not rely on the redirect

* Appeasing meta checker

* Remove dupe meta

* meta correction

* fixing a few things that come up as warnings on the CI

* adding meta

* Fix broken link and hide nav menu thats not used yet

dankeogh99 committed:
Updating the contents of /etc/rabbitmq/rabbitmq-env.conf (#622)

Rabbit refused to start with the original /etc/rabbitmq/rabbitmq-env.conf file.

darrentayloruk committed:
Updating populator to check for existing of the correct meta title (#624)

Updating the test.sh to validate the right meta title
Adding in clearer warning and Fail messages
Updating tests to check for shorter meta as ? UKFast Documentation will be appended going forward to the template

miff2000 committed:
Add TLS to docs.ukfast.co.uk ingress (#623)

* Add new certificate request file

* Add issuer config

* Add TLS listener for docs.ukfast.co.uk

* Delete issuer.yml

Delete `issuer.yml` as it's not longer needed

* Delete certificate.yml

Delete certificate.yml as not used anymore

darrentayloruk committed:
Correcting test, the 2 preceding spaces are super important for this to work (#626)

darrentayloruk committed:
Moving email into its own section and linking to from the homepage (#625)

* Moving email into its own section and linking to from the homepage
* Change any link references and adding in redirects to nginx config

Co-authored-by: Matt Calvert <miff2000@users.noreply.github.com>

darrentayloruk committed:
Some updates to the readme to help clarify some of the meta adding points (#627)

* Updating `readme.md` to help future selves with the crazy meta extracting and precise setup you need to do :)
* Correct indentation for `eval_rst` blocks in `readme.md`

Co-authored-by: Matt Calvert <miff2000@users.noreply.github.com>

miff2000 committed:
Use <pre> block to disable ```eval_rst interpretation

miff2000 committed:
remove eval_rst for none markdown files

miff2000 committed:
Switch from Markdown table formatting to RST

Spotted this page <https://docs.ukfast.co.uk/security/ddos/setup-cname.html> uses a markdown table which doesn't render properly in ReStructuredText, so fixing it

darrentayloruk committed:
Bulk meta move (#628)

* Bulk updating meta titles, removing `| UKFast Documentation`
* Add missing block close marker
* Separate concatenated words

Co-authored-by: Matt Calvert <miff2000@users.noreply.github.com>

darrentayloruk committed:
Append fixed suffix to meta title (#629)

* Put in in here so doesn't have to be in every file

miff2000 committed:
Remove `.html` ending from links. Sphinx handles it (#630)

* Remove `.html` ending from links. Sphinx handles it
* Corrected missing or incomplete metadata blocks

AmberEden committed:
Add site performance docs and images (#631)

* site performance docs - new

* Update optimisewordpress.md

* Update optimisewordpress.md

* Update optimisewordpress.md

Co-authored-by: Amber Eden <amber.eden95@gmail.com>

miff2000 committed:
This removes the unnecessary structure around hmail images (#632)

miff2000 committed:
Sort sphinx build warnings (#633)

* Add an example of using RST tables to readme
* A massive batch of changes to fix the Sphinx errors which appear whenever a Sphinx build takes place.

Commit:
Fix eval_rst blocks

miff2000 committed:
Another big PR to fix formatting (#634)

* * Remove leading spaces on `.. node` and other blocks.
* Fix URLs rendered incorrectly in `eval_rst` blocks.
* Fix code blocks in `eval_rst` blocks.
* Fix various other issues with `eval_rst` blocks.

* fix typo

* Further couple of bits

* Last one

* Update readme to include more help

* Fix missing title

* Further Sphinx fixes

Messier71 committed:
Adding eCloud public doc (#578)

* Adding eCloud public VM creation doc

* Add files via upload

* Fixing links and images

* Added edit CPU doc

* Adding edit CPU images

* Adding edit mem doc

* Edited formatting

* formatting again

* Update index

* Added edit disks doc

* Adding more emphasis

* Formatting

* New line adding

* Updating index file

* Typo

* Adding delete VM and modifying edit disks

* Add files via upload

* Updated index

* Added reboots page

* Add files via upload

* Changing layout

* Adding creds retreival doc

* Add files via upload

* Adding console doc

* Add files via upload

* Adding reinstall doc

* Added to index

* Adding marketplace doc

* Add files via upload

* Add files via upload

* Changes requested in pull made
1: Word added
2: Changed tense
3: Removed not needed file

* Trying to fix meta tags

* Adding "```eval_rst"

* Adding ```eval_rst to rest

* Adding ```eval_rst to apis

* Changing source/cloud to source/ecloud

* Removing head files

* Adding title to correct section of rst

* Removing old cloud directory replaced with ecloud and removing | UKFast Documentation

* Fixing an old cPanel doc that had the wrong rst layout

Co-authored-by: root <root@hznr.benmulpeter.co.uk>
Co-authored-by: Count Bacula <37558685+CountBacula@users.noreply.github.com>
Co-authored-by: root <root@hznr-cloud.benmulpeter.co.uk>

darrentayloruk committed:
little bit of tidy up to improve site nav titles (#635)

darrentayloruk committed:
eCloud hybrid remove (#636)

* Removing the hybrid references
* Adding in redirect for the folder
* Adding meta/change case
* Delete Thumbs.db
Co-authored-by: Matt Calvert <miff2000@users.noreply.github.com>

darrentayloruk committed:
couple of things missed from the rename, index files (#637)

Commit:
Updated the populator to sanitise text going into ElasticSearch better (#639)

* Remove new line chars from elastic search content and strip prepending whitespace from elastic search titles.
* Updated wording
* Minor syntax change to populator.

Co-authored-by: tom.couser <tom.couser@ukfast.co.uk>

miff2000 committed:
Add new guide on ESXi upgrades (#640)

* Add new guide on ESXi upgrades
* Add to Operating Systems TOC
* Add VMware to the nav bar

Messier71 committed:
Adding new eCloud marketplace appliance config screenshot with new pod locations (#641)

Co-authored-by: Ben Mulpeter <ben.mulpeter@ukfast.co.uk>

darrentayloruk committed:
CSS and Layout updates to show breadcrumb navigation to help site usability (#643)

darrentayloruk committed:
2 Bugs in populator (#642)

* 1 Title search not quite right (my fault in PR #624) missing a 2nd :
* 2 Noticed search results broken link

`/ecloud/flex/resources/network/index.html`
was returned as
`/ecloud/flex/res/network/index.html`

Found 'source' being replaced in the populator - doesn't appear to be required

darrentayloruk committed:
Extract the current pages navigation text from the TOC and use for the current page breadcrumb instead of the page.title (which pulls meta title) (#646)

add ecommerce stacks into homepage boxes
Added example nav override into eCloud flex

darrentayloruk committed:
Update readme with details on title, meta (#647)

Providing detail on how sections are used and where so its more clear where content is used

darrentayloruk committed:
domain name FAQs (#648)

AmberEden committed:
New doc on connection to mysql remotely (#650)

* Create MySQL Remote doc

Create MySQL Remote doc

* Update index.rst

add new docs page

32bitbradley committed:
Updated Threat Monitoring Install docs (#649)

* Order change

Edited order to be more logical

* Added non UKfast install docs

Added non UKfast install docs

* Added ukfast hosted docs

Added ukfast hosted docs

* Add files via upload

* Delete setup-centos.png

* Add files via upload

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Removed ** from link

Removed ** from link causing it not to resolve

* Updated Control headings

Updated Control headings to be bolder with new design

* Update thmon-for-pci-dss.md

* Create system-requirements.md

Added system requirements

* Update system-requirements.md

* Update system-requirements.md

* Update system-requirements.md

* Add files via upload

* Update ukfast-hosted-install.md

* Update ukfast-hosted-install.md

* Add files via upload

* Update ukfast-hosted-install.md

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

* Update thmon-for-pci-dss.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

* Update thmon-for-pci-dss.md

* Update system-requirements.md

* Update ukfast-hosted-install.md

* Update non-ukfast-install.md

* Update safedns-setup.md

* Update system-requirements.md

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

* Update index.rst

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

* Update index.rst

* Update index.rst

* Update non-ukfast-install.md

* Update system-requirements.md

* Rename thmon-for-pci-dss.md to pci-dss.md

* Update index.rst

* Update ukfast-hosted-install.md

* Update index.rst

* Update non-ukfast-install.md

* Add files via upload

* Update getting-started.md

* Update getting-started.md

* Update getting-started.md

32bitbradley committed:
System Requirements Link Fixed (#651)

* Order change

Edited order to be more logical

* Added non UKfast install docs

Added non UKfast install docs

* Added ukfast hosted docs

Added ukfast hosted docs

* Add files via upload

* Delete setup-centos.png

* Add files via upload

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Removed ** from link

Removed ** from link causing it not to resolve

* Updated Control headings

Updated Control headings to be bolder with new design

* Update thmon-for-pci-dss.md

* Create system-requirements.md

Added system requirements

* Update system-requirements.md

* Update system-requirements.md

* Update system-requirements.md

* Add files via upload

* Update ukfast-hosted-install.md

* Update ukfast-hosted-install.md

* Add files via upload

* Update ukfast-hosted-install.md

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

* Update thmon-for-pci-dss.md

* Update non-ukfast-install.md

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

* Update thmon-for-pci-dss.md

* Update system-requirements.md

* Update ukfast-hosted-install.md

* Update non-ukfast-install.md

* Update safedns-setup.md

* Update system-requirements.md

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

* Update index.rst

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

* Update index.rst

* Update index.rst

* Update non-ukfast-install.md

* Update system-requirements.md

* Rename thmon-for-pci-dss.md to pci-dss.md

* Update index.rst

* Update ukfast-hosted-install.md

* Update index.rst

* Update non-ukfast-install.md

* Add files via upload

* Update getting-started.md

* Update getting-started.md

* Update getting-started.md

* Update non-ukfast-install.md

* Update ukfast-hosted-install.md

darrentayloruk committed:
Adding (back) a source replacement in the URLs, but this time only at the start. (#654)

Fixes odd case where enter keypress on search result even uses this URL instead of the url presented in the search results

DylanHBL committed:
Updating build content. (#609)

* Update enterprise_sql_default.md

* Update software.md

* Update partition.md

* Update web_default.md

* Update web_default.md

* Update file_default.md

* Update mysql_default.md

* Update enterprise_sql_default.md

* Update file_default.md

* Update mysql_default.md

* Update partition.md

* Update software.md

* Update web_default.md

* Update enterprise_sql_default.md

* Update enterprise_sql_default.md

* Update mysql_default.md

* Update partition.md

* Update software.md

* Update web_default.md

* Update file_default.md

terencesimms committed:
Let's Encrypt Documentation (#655)

* Let's Encrypt Doc for Linux

* Let's Encrypt - Section on Windows ACME clients

* Let's Encrypt - few changes to posh-acme and ratelimits

* removing pdf

* Update letsencrypt.md

amending docs links to other docs

* Update index.rst

Adding letsencrypt and updating the toc tree depth

* Update letsencrypt.md

adding a forward slash to operatingsystems/linux/basics/connecting

Co-authored-by: Terence Simms <Terence.Simms@support.ukfast>

darrentayloruk committed:
Overriding Nav (#656)

Having to making linux and windows a h2 as affects nav

AmberEden committed:
New docs for restoring and migrating a database (#657)

Co-authored-by: amber eden <amber.eden@ukfast.co.uk>

AmberEden committed:
Fix a broken doc referrer (#658)

* New docs for restoring and migrating a database

* Fix a broken doc referrer 

Fix a broken doc referrer

* Fix a broken doc referrer 

Fix a broken doc referrer

Co-authored-by: amber eden <amber.eden@ukfast.co.uk>

terencesimms committed:
Letsencrypt v2 (#659)

* Update letsencrypt.md

amending typo.

* changing a link from a linked link to a full link

terencesimms committed:
Update cpanel_add_account.md (#660)

Adding a section on what an account contains, and a warning about adding a domain that matches the hostname as will break things

terencesimms committed:
Letsencryptv3 (#661)

* Update index.rst

Adding apostrophe to Let's Encrypt in TOC

* Update letsencrypt.md

Amending meta title to match other SSL guides, and adding 'letsencrypt' to the keywords for search

* Update letsencrypt.md

Changing the title of the doc and H1

chrotek committed:
Added docs for Fixing deceptive site warning for WHM in Chrome (#662)

* Added docs for Fixing deceptive site warning for WHM in Chrome

Added docs for Fixing deceptive site warning for WHM in Chrome

* Update cpanel_fixdeceptivesitewarning.md

added metadata

darrentayloruk committed:
Fixing opensearch xml to reference docs (#664)

darrentayloruk committed:
remove source replace (#666)

tom-taylor committed:
Added 7.4 (#669)

tom-taylor committed:
Added PHP 7.4 (#668)

george-fr committed:
Adding myukfast docs page (#667)

* Create managing_contacts.md

* Update index.rst

* Create index.rst

* Create account-manager.md

* Update index.rst

* Create accounts.md

* Update index.rst

* Create my_company_details.md

* Update index.rst

* Update my_company_details.md

* Rename account-manager.md to account_manager.md

* Update index.rst

* Update index.rst

* Rename account_manager.md to account-manager.md

* Update accounts.md

* Update my_company_details.md

* Update and rename accounts.md to billing.md

* Update account-manager.md

* Update managing_contacts.md

* Update index.rst

* Rename managing_contacts.md to managing-contacts.md

* Update and rename my_company_details.md to my-company-details.md

* Update billing.md

* Update billing.md

darrentayloruk committed:
fix myukfast index page (#670)

link tweak

AmberEden committed:
New New-relic doc (#672)

tom-taylor committed:
Added option for PHP version (#675)

tom-taylor committed:
added option for PHP version (#676)

tom-taylor committed:
corrected a spelling mistake (#677)

adamskie committed:
ecommerce patch 1 (#674)

* Update rabbitmq.md
* Update varnish.md

tom-taylor committed:
Saying how 7 can be installed rather than 6 (#678)

* Saying how 7 can be installed rather than 6

* Update elasticsearch.md

* Update elasticsearch.md

* Update elasticsearch.md

Co-authored-by: adamskie <adamskie@gmail.com>

adamcade committed:
Update index.rst (#679)

XargsUK committed:
Update installnewrelic.md as they have created a free tier option (#681)

* Update installnewrelic.md docs to reflect pricing structure change

benabean committed:
Update wordpress.md (#682)

Updated links that no longer worked

thatdamndog committed:
Updating envoy console page (#692)

Added screenshots for both eCloud private and public. Made page generic and added links from both public and private indexes

benabean committed:
Update billing.md (#691)

Moved metadata to the bottom of the document

benabean committed:
Update my-company-details.md (#690)

Moved metadata to the bottom of the document

benabean committed:
Update managing-contacts.md (#689)

* Moved meta data to the bottom of document
* Added in section about controlling contact access to account

danclare committed:
New doc on connecting to MSSQL remotely (#693)

* Create mssql_remote.md

0x4c6565 committed:
Update .gitlab-ci.yml

benabean committed:
Create managing-passwords.md (#685)

* Create managing-passwords.md

Information on managing passwords within MyUKFast and what each type of password is used for

* Update managing-passwords.md

Added in hyperlinkks [MyUKFast](https://www.my.ukfast.co.uk)

* Update managing-passwords.md

* Create placeholder.txt

Added in files for screenshots

* Add files via upload

* Add files via upload

* Update managing-passwords.md

Added in screenshots

* Update managing-passwords.md

adding a warning tag at the bottom, and subheadings for password types

Co-authored-by: terencesimms <tardismeat@gmail.com>

dansjackson committed:
Updates to New Relic Docs (#700)

* added guide on how to define PHP install path

* changed the layout of the information

* made the KB more accessible

* more formatting fixes

* Update installnewrelic.md

amending service commands, adding panel note about restarting services, removing nginx restart (no need) & tweaking a sentence, plus example subheadings for linkability

Co-authored-by: terencesimms <tardismeat@gmail.com>

benabean committed:
Update index.rst (#686)

* Update index.rst

add in managing passwords page

* Update index.rst

amending spelling of 'access' from 'accesses'

Co-authored-by: terencesimms <tardismeat@gmail.com>

tom-taylor committed:
Rearranged the HTTPS redirect section (#701)

Made that section more readable and added the command to config test and reload Varnish

tom-taylor committed:
Changed service .. reload to systemctl (#703)

terencesimms committed:
Update installnewrelic.md (#702)

- Changing the title to remove php agent, as implied
- Updating the meta data to reflect this title and appear in search results
- Updating numbered sequence to bullet points, as numbers don't persist between code blocks and defaults to 1. again.
- Switching around parts of the agent installation, and clarifying where to get the APM key from with a link.
- Adding a troubleshooting section on New Relic sock file issues.

terencesimms committed:
Letsencryptv4 (#694)

* creating letsencrypt dirs

* Create index.rst

* Create letsencrypt_how.md

* Create letsencrypt_centos_nginx.md

* apache/plesk/ubuntu/cpanel/windows/autorenew

adding the above docs

* deleting - has been split in the letsencrypt folder

* updating to reflect new letsencrypt toc

* Update index.rst

amending letsencrypt/index link, as unsure if the < > version works the same

* Delete placeholder.md

* Create placeholder.md

* Adding plesk/cpanel screenshots

* Delete placeholder.md

* Delete cpanel_autossl_letsencrypt_enable.PNG

moved to letsencrypt folder

* Delete cpanel_autossl_logs.PNG

moved to letsencrypt folder

* Delete cpanel_autossl_manageusers.PNG

moved to letsencrypt folder

* Delete cpanel_autossl_search.PNG

moved to letsencrypt folder

* Delete plesk_letsencrypt_existingdomain.PNG

moved to letsencrypt folder

* Delete plesk_letsencrypt_extension.PNG

moved to letsencrypt folder

* Delete plesk_letsencrypt_newdomain.PNG

moved to letsencrypt folder

* Delete plesk_letsencrypt_services.PNG

moved to letsencrypt folder

* updating links for apache/nginx in ubuntu doc

* Update letsencrypt_autorenew_revoke.md

amending spacing and capitalisation

* Update index.rst

deleting meta content

* Update index.rst

removing additional toctrees

terencesimms committed:
Letsencrypt5 (#705)

* Lets Encrypt typos, adding powershell linting and a note about project contributions.

Commit:
Update Fork

Commit:
Merge remote-tracking branch 'origin/master'
